### PR TITLE
detect all '<' characters

### PR DESCRIPTION
**Pull request by 70ray**
and mark as possible unrecognised tags, encode &, <, >  so output is valid html and browsers do not leave anything out.
This addresses the issue found in https://www.pgdp.net/phpBB3/viewtopic.php?f=34&t=61794&p=1273703#p1273703

### CODE REVIEW

**Comment by srjfoo** [comment]
@70ray, have you got a sandbox for testing?

**Comment by 70ray** [comment]
> @70ray, have you got a sandbox for testing?

Forgot. Done it now.
Sandbox at: https://www.pgdp.org/~rp31/c.branch/tag_check

File: SETUP/tests/jsTests/formatPreviewTests.js [hunk 1]
```diff
@@ -102,8 +102,8 @@ QUnit.module("Format preview test", function() {
     QUnit.test("unrecognised tag, u not enabled", function (assert) {
```
Comment (chrismiceli) [inline SETUP/tests/jsTests/formatPreviewTests.js:102]: Can we add some tests similar to the issue documented in the forums with the makePreview function?

Comment (70ray) [reply SETUP/tests/jsTests/formatPreviewTests.js:102]: Good idea, I'll look into it.

**Review by srjfoo — APPROVED** [review]

**Pull request merged**